src/manager.c

```diff
@@ -40,215 +40,13 @@
 #include "settings.h"
 #include "msg.h"
```

Comment:
About commit msg 071f724d0b28b6827cb5c326d01a173700697640:
... by **the** AMQP ...

Reply:
Ok